app/controllers/episodes_controller.rb

```diff
@@ -92,7 +92,7 @@ def destroy
 
     respond_to do |format|
       if @episode.destroy
-        @episode.publish!
+        @episode.podcast.publish!
```

Comment:
Are you trying to skip the `apple_mark_for_reupload!` when the episode is deleted?  It might be better to add a `unless deleted?` check to that episode method then, in case someone calls `episode.publish!` on a deleted episode from somewhere else in the codebase.

Reply:
Patched that in ccc9c62d7eb0f286b0896a34d2c992381c01071d